---
layout: infopage
title: Publications
---
### 2023

Blesl A, Wurm P, <u>Waschina S</u>, Gröchenig HP, Novacek G, Primas C, Reinisch W, Kutschera M, Illiasch C, Hennlich B, Steiner P, Koch R, Tillinger W, Haas T, Reicht G, Mayer A, Ludwiczek O, Miehsler W, Steidl K, Binder L, Reider S, Watschinger C, Fürst S, Kump P, Moschen A, Aden K, Gorkiewicz G, Högenauer C.<br>[**Prediction of Response to Systemic Corticosteroids in Active UC by Microbial Composition—A Prospective Multicenter Study**](http://dx.doi.org/10.1093/ibd/izad126)<br>*Inflammatory Bowel Diseases*. 2023. . doi: 10.1093/ibd/izad126

Effenberger M, <u>Waschina S</u>, Bronowski C, Sturm G, <u>Tassiello O</u>, Sommer F, Zollner A, Watschinger C, Grabherr F, Gstir R, Grander C, Enrich B, Bale R, Putzer D, Djanani A, Moschen AR, Zoller H, Rupp J, Schreiber S, Burcelin R, Lass-Flörl C, Trajanoski Z, Oberhuber G, Rosenstiel P, Adolph TE, Aden K, Tilg H.<br>[**A gut bacterial signature in blood and liver tissue characterizes cirrhosis and hepatocellular carcinoma**](http://dx.doi.org/10.1097/HC9.0000000000000182)<br>*Hepatology Communications*. 2023. 7(7). doi: 10.1097/hc9.0000000000000182

<u>Busche S</u>, <u>Harris DMM</u>, Zimmermann J, Schuchardt S, Oumari M, Frank D, Bang C, Rosenstiel P, Schreiber S, Frey N, Franke A, Aden K, <u>Waschina S</u>.<br>[**Amino acid auxotrophies in human gut bacteria are linked to higher microbiome diversity and long-term stability**](http://dx.doi.org/10.1101/2023.03.23.532984)<br>(preprint) *bioRxiv*. 2023. doi: 10.1101/2023.03.23.532984

Rühlemann MC, Bang C, Gogarten JF, Hermes BM, Groussin M, <u>Waschina S</u>, Poyet M, Ulrich M, Akoua-Koffi C, Deschner T, Muyembe-Tamfum JJ, Robbins MM, Surbeck M, Wittig RM, Zuberbühler K, Baines JF, Leendertz FH, Franke A.<br>[**Comparative metagenomics reveals host-specific functional adaptation of intestinal microbiota across hominids**](http://dx.doi.org/10.1101/2023.03.01.530589)<br>(preprint) *bioRxiv*. 2023. doi: 10.1101/2023.03.01.530589

Marinos G, Hamerich IK, Debray R, Obeng N, Petersen C, Taubenheim J, Zimmermann J, Blackburn D, Samuel BS, Dierking K, Franke A, Laudes M, <u>Waschina S</u>, Schulenburg H, Kaleta C.<br>[**Metabolic model predictions enable targeted microbiome manipulation through precision prebiotics**](http://dx.doi.org/10.1101/2023.02.17.528811)<br>(preprint) *bioRxiv*. 2023. doi: 10.1101/2023.02.17.528811

### 2022

Jensen-Kroll J, Demetrowitsch T, Clawin-Rädecker I, Klempt M, <u>Waschina S</u>, Schwarz K.<br>[**Microbiota independent effects of oligosaccharides on Caco-2 cells -A semi-targeted metabolomics approach using DI-FT-ICR-MS coupled with pathway enrichment analysis**](http://dx.doi.org/10.3389/fmolb.2022.968643)<br>*Frontiers in Molecular Biosciences*. 2022. 9. doi: 10.3389/fmolb.2022.968643

<u>Waschina S</u>, Seeger K.<br>[**Using in-tube extraction and slice selective NMR experiments allow imaging via statistical analysis of metabolic profiles**](http://dx.doi.org/10.1016/j.aca.2022.340419)<br>*Analytica Chimica Acta*. 2022. 1231. doi: 10.1016/j.aca.2022.340419

Troci A, Rausch P, <u>Waschina S</u>, Lieb W, Franke A, Bang C.<br>[**Long‐Term Dietary Effects on Human Gut Microbiota Composition Employing Shotgun Metagenomics Data Analysis**](http://dx.doi.org/10.1002/mnfr.202101098)<br>*Molecular Nutrition &amp; Food Research*. 2022. . doi: 10.1002/mnfr.202101098

Schwenker JA, Friedrichsen M, <u>Waschina S</u>, Bang C, Franke A, Mayer R, Hölzel CS.<br>[**Bovine milk microbiota: Evaluation of different DNA extraction protocols for challenging samples**](http://dx.doi.org/10.1002/mbo3.1275)<br>*MicrobiologyOpen*. 2022. 11(2). doi: 10.1002/mbo3.1275

Bossung V, Lupatsii M, Dashdorj L, <u>Tassiello O</u>, Jonassen S, Pagel J, Demmert M, Wolf EA, Rody A, <u>Waschina S</u>, Graspeuntner S, Rupp J, Härtel C.<br>[**Timing of antimicrobial prophylaxis for cesarean section is critical for gut microbiome development in term born infants**](http://dx.doi.org/10.1080/19490976.2022.2038855)<br>*Gut Microbes*. 2022. 14(1). doi: 10.1080/19490976.2022.2038855

### 2021

Giri S, Oña L, <u>Waschina S</u>, Shitut S, Yousif G, Kaleta C, Kost C.<br>[**Metabolic dissimilarity determines the establishment of cross-feeding interactions in bacteria**](http://dx.doi.org/10.1016/j.cub.2021.10.019)<br>*Current Biology*. 2021. 31(24). doi: 10.1016/j.cub.2021.10.019

Schönborn JW, Stewart FA, Enriquez KM, Akhtar I, Droste A, <u>Waschina S</u>, Beller M.<br>[**Modeling Drosophila gut microbe interactions reveals metabolic interconnectivity**](http://dx.doi.org/10.1016/j.isci.2021.103216)<br>*iScience*. 2021. 24(11). doi: 10.1016/j.isci.2021.103216

Josephs-Spaulding J, Krogh TJ, Rettig HC, Lyng M, Chkonia M, <u>Waschina S</u>, Graspeuntner S, Rupp J, Møller-Jensen J, Kaleta C.<br>[**Recurrent Urinary Tract Infections: Unraveling the Complicated Environment of Uncomplicated rUTIs**](http://dx.doi.org/10.3389/fcimb.2021.562525)<br>*Frontiers in Cellular and Infection Microbiology*. 2021. 11. doi: 10.3389/fcimb.2021.562525

Zimmermann J, Kaleta C, <u>Waschina S</u>.<br>[**gapseq: informed prediction of bacterial metabolic pathways and reconstruction of accurate metabolic models**](http://dx.doi.org/10.1186/s13059-021-02295-1)<br>*Genome Biology*. 2021. 22(1). doi: 10.1186/s13059-021-02295-1

Schilf P, Künstner A, Olbrich M, <u>Waschina S</u>, Fuchs B, Galuska CE, Braun A, Neuschütz K, Seutter M, Bieber K, Hellberg L, Sina C, Laskay T, Rupp J, Ludwig RJ, Zillikens D, Busch H, Sadik CD, Hirose M, Ibrahim SM.<br>[**A Mitochondrial Polymorphism Alters Immune Cell Metabolism and Protects Mice from Skin Inflammation**](http://dx.doi.org/10.3390/ijms22031006)<br>*International Journal of Molecular Sciences*. 2021. 22(3). doi: 10.3390/ijms22031006

### 2020

Effenberger M, Reider S, <u>Waschina S</u>, Bronowski C, Enrich B, Adolph TE, Koch R, Moschen AR, Rosenstiel P, Aden K, Tilg H.<br>[**Microbial Butyrate Synthesis Indicates Therapeutic Efficacy of Azathioprine in IBD Patients**](http://dx.doi.org/10.1093/ecco-jcc/jjaa152)<br>*Journal of Crohn's and Colitis*. 2020. 15(1). doi: 10.1093/ecco-jcc/jjaa152

Demetrowitsch TJ, Schlicht K, Knappe C, Zimmermann J, Jensen-Kroll J, <u>Pisarevskaja A</u>, Brix F, Brandes J, Geisler C, Marinos G, Sommer F, Schulte DM, Kaleta C, Andersen V, Laudes M, Schwarz K, <u>Waschina S</u>.<br>[**Precision Nutrition in Chronic Inflammation**](http://dx.doi.org/10.3389/fimmu.2020.587895)<br>*Frontiers in Immunology*. 2020. 11. doi: 10.3389/fimmu.2020.587895

Pagel J, Twisselmann N, Rausch TK, <u>Waschina S</u>, Hartz A, Steinbeis M, Olbertz J, Nagel K, Steinmetz A, Faust K, Demmert M, Göpel W, Herting E, Rupp J, Härtel C.<br>[**Increased Regulatory T Cells Precede the Development of Bronchopulmonary Dysplasia in Preterm Infants**](http://dx.doi.org/10.3389/fimmu.2020.565257)<br>*Frontiers in Immunology*. 2020. 11. doi: 10.3389/fimmu.2020.565257

Bhat J, Bergmann AK, <u>Waschina S</u>, Nerl C, Kaleta C, Siebert R, Ammerpohl O, Kabelitz D.<br>[**DNA methylation profile of a hepatosplenic gamma/delta T-cell lymphoma patient associated with response to interferon-α therapy**](http://dx.doi.org/10.1038/s41423-020-0518-4)<br>*Cellular &amp; Molecular Immunology*. 2020. 18(5). doi: 10.1038/s41423-020-0518-4

Marinos G, Kaleta C, <u>Waschina S</u>.<br>[**Defining the nutritional input for genome-scale metabolic models: A roadmap**](http://dx.doi.org/10.1371/journal.pone.0236890)<br>*PLOS ONE*. 2020. 15(8). doi: 10.1371/journal.pone.0236890

### 2019

Aden K, Rehman A, <u>Waschina S</u>, Pan WH, Walker A, Lucio M, Nunez AM, Bharti R, Zimmerman J, Bethge J, Schulte B, Schulte D, Franke A, Nikolaus S, Schroeder JO, Vandeputte D, Raes J, Szymczak S, Waetzig GH, Zeuner R, Schmitt-Kopplin P, Kaleta C, Schreiber S, Rosenstiel P.<br>[**Metabolic Functions of Gut Microbes Associate With Efficacy of Tumor Necrosis Factor Antagonists in Patients With Inflammatory Bowel Diseases**](http://dx.doi.org/10.1053/j.gastro.2019.07.025)<br>*Gastroenterology*. 2019. 157(5). doi: 10.1053/j.gastro.2019.07.025

Giri S, <u>Waschina S</u>, Kaleta C, Kost C.<br>[**Defining Division of Labor in Microbial Communities**](http://dx.doi.org/10.1016/j.jmb.2019.06.023)<br>*Journal of Molecular Biology*. 2019. 431(23). doi: 10.1016/j.jmb.2019.06.023

Zimmermann J, Obeng N, Yang W, Pees B, Petersen C, <u>Waschina S</u>, Kissoyan KA, Aidley J, Hoeppner MP, Bunk B, Spröer C, Leippe M, Dierking K, Kaleta C, Schulenburg H.<br>[**The functional repertoire contained within the native microbiota of the model nematode Caenorhabditis elegans**](http://dx.doi.org/10.1038/s41396-019-0504-y)<br>*The ISME Journal*. 2019. 14(1). doi: 10.1038/s41396-019-0504-y

Pryor R, Norvaisas P, Marinos G, Best L, Thingholm LB, Quintaneiro LM, De Haes W, Esser D, <u>Waschina S</u>, Lujan C, Smith RL, Scott TA, Martinez-Martinez D, Woodward O, Bryson K, Laudes M, Lieb W, Houtkooper RH, Franke A, Temmerman L, Bjedov I, Cochemé HM, Kaleta C, Cabreiro F.<br>[**Host-Microbe-Drug-Nutrient Screen Identifies Bacterial Effectors of Metformin Therapy**](http://dx.doi.org/10.1016/j.cell.2019.08.003)<br>*Cell*. 2019. 178(6). doi: 10.1016/j.cell.2019.08.003

Yang W, Petersen C, Pees B, Zimmermann J, <u>Waschina S</u>, Dirksen P, Rosenstiel P, Tholey A, Leippe M, Dierking K, Kaleta C, Schulenburg H.<br>[**The Inducible Response of the Nematode Caenorhabditis elegans to Members of Its Natural Microbiota Across Development and Adult Life**](http://dx.doi.org/10.3389/fmicb.2019.01793)<br>*Frontiers in Microbiology*. 2019. 10. doi: 10.3389/fmicb.2019.01793

### 2018

Graspeuntner S, <u>Waschina S</u>, Künzel S, Twisselmann N, Rausch TK, Cloppenborg-Schmidt K, Zimmermann J, Viemann D, Herting E, Göpel W, Baines JF, Kaleta C, Rupp J, Härtel C, Pagel J.<br>[**Gut Dysbiosis With Bacilli Dominance and Accumulation of Fermentation Products Precedes Late-onset Sepsis in Preterm Infants**](http://dx.doi.org/10.1093/cid/ciy882)<br>*Clinical Infectious Diseases*. 2018. 69(2). doi: 10.1093/cid/ciy882

D'Souza G, Shitut S, Preussger D, Yousif G, <u>Waschina S</u>, Kost C.<br>[**Ecology and evolution of metabolic cross-feeding interactions in bacteria**](http://dx.doi.org/10.1039/c8np00009c)<br>*Natural Product Reports*. 2018. 35(5). doi: 10.1039/c8np00009c

### 2016

<u>Waschina S</u>, D'Souza G, Kost C, Kaleta C.<br>[**Metabolic network architecture and carbon source determine metabolite production costs**](http://dx.doi.org/10.1111/febs.13727)<br>*The FEBS Journal*. 2016. 283(11). doi: 10.1111/febs.13727

Gebauer J, Gentsch C, Mansfeld J, Schmeißer K, <u>Waschina S</u>, Brandes S, Klimmasch L, Zamboni N, Zarse K, Schuster S, Ristow M, Schäuble S, Kaleta C.<br>[**A Genome-Scale Database and Reconstruction of Caenorhabditis elegans Metabolism**](http://dx.doi.org/10.1016/j.cels.2016.04.017)<br>*Cell Systems*. 2016. 2(5). doi: 10.1016/j.cels.2016.04.017

### 2015

D'Souza G, <u>Waschina S</u>, Kaleta C, Kost C.<br>[**Plasticity and epistasis strongly affect bacterial fitness after losing multiple metabolic genes**](http://dx.doi.org/10.1111/evo.12640)<br>*Evolution*. 2015. 69(5). doi: 10.1111/evo.12640

### 2014

Schmidt R, <u>Waschina S</u>, Boettger-Schmidt D, Kost C, Kaleta C.<br>[**Computing autocatalytic sets to unravel inconsistencies in metabolic network reconstructions**](http://dx.doi.org/10.1093/bioinformatics/btu658)<br>*Bioinformatics*. 2014. 31(3). doi: 10.1093/bioinformatics/btu658

D'Souza G, <u>Waschina S</u>, Pande S, Bohl K, Kaleta C, Kost C.<br>[**LESS IS MORE: SELECTIVE ADVANTAGES CAN EXPLAIN THE PREVALENT LOSS OF BIOSYNTHETIC GENES IN BACTERIA**](http://dx.doi.org/10.1111/evo.12468)<br>*Evolution*. 2014. 68(9). doi: 10.1111/evo.12468

